src/main/java/com/akademia/projectplanner/entity/UserEntity.java

```diff
@@ -3,16 +3,20 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
 
 import javax.persistence.*;
+import java.util.Collection;
 import java.util.List;
 
 @NoArgsConstructor
 @Getter
 @Setter
 @Entity
 @Table(name = "users")
-public class UserEntity {
+public class UserEntity implements UserDetails {
```

Comment:
W Encjach nie implementujemy nic, to jest obiekt który jest bezpośrednią reprezentacją wiersza w bazie danych. Dobrze kombinujesz ale podobny interfejs (UserDetailsService) dodamy do serwisu użytkownika gdzie będziemy wyciągać dane do logowania